fl4health/datasets/skin_cancer/preprocess_skin.py

```diff
@@ -0,0 +1,281 @@
+# The following code is adapted from the preprocess_skin.py script
+# from the medical_federated GitHub repository by Seongjun Yang et al.
+# Paper: https://arxiv.org/abs/2207.03075
+# Code: https://github.com/wns823/medical_federated.git
+# - medical_federated/skin_cancer_federated/preprocess_skin.py
+
+import json
+import os
+from typing import Any, Dict
+
+import pandas as pd
+
+
+def save_to_json(data: Dict[str, Any], path: str) -> None:
+    with open(path, "w", encoding="utf-8") as file:
+        json.dump(data, file, indent="\t")
+
+
+if __name__ == "__main__":
+
+    data_path = "fl4health/datasets/skin_cancer"
+
+    # Load GT file of ISIC-19
+    ISIC_2019_path = f"{data_path}/ISIC_2019"
```

Comment:
Rather than using f-strings, the convention is typically to use `os.path.join` or define a Path object and join that way 🙂

Reply:
There are a few other places that we could clean this up below as well

Reply:
os.path.join 's all the way!